roles/openshift_master/tasks/main.yml

```diff
@@ -12,7 +19,9 @@
     dest: /etc/sysconfig/openshift-master
     regexp: "{{ item.regex }}"
     line: "{{ item.line }}"
+    create: yes
   with_items:
+    - { regex: '^ROLE=', line: 'ROLE=\"master\"' }
```

Comment:
This line is now pre-populated from the openshift-master and openshift-node packages.